packages/shared/styles/components/atoms/SfButton.scss

```diff
@@ -134,13 +116,24 @@
           --button-color: var(--c-gray);
           --button-background: transparent;
         }
-        &:disabled {
-          &:hover {
-            --button-color: var(--c-text-muted);
-          }
-        }
       }
     }
+  }  
+  &.is-disabled--button,
+  &.is-disabled--button:hover {
+    --button-color: var(--c-text-disabled);
```

Comment:
can you try setting  `pointer-events: none` to this and see if it works, instead of setting the link to `null`? Thanks @AdamPawlinski

Reply:
Yes of course.

Reply:
Looks like it works good, thanks @mayashavin and @justyna-13 :)